src/cli/bootupctl.rs

```diff
@@ -228,3 +216,15 @@ fn run_status_in_container(json_format: bool) -> Result<()> {
     }
     Ok(())
 }
+
+/// Initialize parent devices to prepare the update
+fn prep_before_update() -> Result<openat::Dir> {
+    let sysroot = openat::Dir::open("/").context("Opening root dir")?;
+    let dest_fd = format!("/proc/self/fd/{}", sysroot.as_raw_fd());
+    let dest_root = std::fs::read_link(dest_fd)?;
```

Comment:
Per above I think we can just for now hardcode something like
`let devices = crate::blockdev::get_devices("/")` here.

It actually may make sense to move the `static` to this function as it's the one that's also hardcoding `/`.